lib/manager/gradle/index.ts

```diff
@@ -29,7 +29,11 @@ export function updateDependency(
 export const language = LANGUAGE_JAVA;
 
 export const defaultConfig = {
-  fileMatch: ['\\.gradle(\\.kts)?$', '(^|/)gradle.properties$'],
+  fileMatch: [
+    '\\.gradle(\\.kts)?$',
+    '(^|/)gradle.properties$',
+    'gradle\\/.+\\.toml$',
```

Comment:
```suggestion
    '(^|\\//gradle\\/.+\\.toml$',
```
So we don't match `xyz-gradle` folders

Reply:
It's not so bad to match them I think. If you want, I can merge the change, just tell me if you are sure.

Reply:
Please do it, because we already do it for the other regexes. 🤗